### PR TITLE
v4.21.0

### PR DESCRIPTION
**Pull request by yotamberk**

### CODE REVIEW

File: misc/RELEASE_CHECKLIST_TEMPLATE.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
  - Keep the order of the commits. Older commits are lower newers are higher.
  - Bug-Fixes start with `FIX #issue:`
```
Comment (wimrijnders) [inline misc/RELEASE_CHECKLIST_TEMPLATE.md]: Missing 'Added' in list.

File: HISTORY.md [hunk 2]
```diff
@@ -2,6 +2,97 @@
...
+- FEAT #824: Network detect clicks on labels
+- FIX #3474: Adjust documentation for arrows.middle.scaleFactor
+- FIX #3483: Prevent image loading for hidden cluster nodes
+- FIX #3408, #2677: Fix handling of multi-fonts
+- FIX #3425: IE performance improvements
+- FIX #3356 and #3297: IE11 svg image fixes
+- FIX #3474: Make negative scaleFactor reverse middle arrow correctly 
+- FIX #3464: Fix handling of space before huge word in label text
+- FIX #3467: Adjust for-in loops so they can deal with added properties in Array and Object prototypes
+- REFA #3442: Strapping down of Extend-routines in util.js
```
Comment (wimrijnders) [inline HISTORY.md]: To 'General'

File: HISTORY.md [hunk 3]
```diff
@@ -2,6 +2,97 @@
...
+- FEAT #3368: Cluster node handling due to dynamic data change
+- FIX #3395: Allow for multiline titles
+- FIX #3367: Network Clustering fixes on usage joinCondition for clusterOutliers()
+- FIX #3350: Fix setting of edge color options via Network.setOptions()
+- FEAT #3348: Add edge styles support for DOT lib
+- FIX #2839: Re-words documentation to reflect symmetrical input/output of get() when passed multiple ids
+- FIX #3316: Updates network documentation to account for edge
+- FIX  #1218, #1291, #1315: Dynamically adjust clustering when data changes
+- FIX #2311: Block recalculation of level in LayoutEngine._determineLevelsDirected()
+- FIX #3280: Cleanup mergeOptions() and fix missing ref on globalOptions in mergeOptions()
```
Comment (wimrijnders) [inline HISTORY.md]: Why italic? Also below.

File: HISTORY.md [hunk 4]
```diff
@@ -2,6 +2,97 @@
...
+- FIX #3126: Nested groups order logic
+- FIX #3246: Fix issue when showMajorLabels == false is used with a weekly scale and weekly scale minor label fix
+- FIX #3342: Bug fix for null parent
+- FIX #2123: Disable the default handling of the pinch event when handling it
+- FIX #3169: add parenthesis around ternary
+- FIX #3249: Only draw non-visible items once when they are loaded, instead of continuously every frame
+- FEAT #3162: Bidirectional scrolling in timeline - make horizontalScroll and verticalScroll work together
+
+
 ## 2017-07-01, version 4.20.1
```
Comment (wimrijnders) [inline HISTORY.md]: Only 3 months since last release? Seems much longer.

File: HISTORY.md (hunk 3 above)
Comment (yotamberk) [reply HISTORY.md]: I have no idea why this is happening... wierd

**Pull request merged**